src/main/java/sirius/biz/storage/layer2/mongo/MongoBlobStorageSpace.java

```diff
@@ -404,6 +404,14 @@ protected void updateBlobName(MongoBlob blob, String newName) {
         mango.update(blob);
     }
 
+    @Override
+    protected void updateBlobReadOnlyFlag(MongoBlob blob, boolean readOnly) {
+        mongo.update()
+             .set(MongoBlob.READ_ONLY, readOnly)
+             .where(MongoBlob.ID, blob.getId())
+             .executeForOne(MongoBlob.class);
```

Comment:
**Optional:** Check if the update result has a count of one and if not log an error like in SQL?

Reply:
In SQL we don't count updated rows, but have to catch for SQLExceptions.
Other Mongo Operations in this class also run in fire-and-forget mode (eg. markBlobAsDeleted). If we change this, than should revise all other calls.